text/0000-at-least-one-feature.md

```diff
@@ -191,14 +192,89 @@ there is no point of adding such a new minimal feature because there is nothing
 This solution is more mathematically elegant, but it seems harder to implement.
 It is unclear how Cargo could require the Rust code to obey this property without new infra like the portability lint.
 
+Finally, in a sense these variations are actually isomorphic.
+If no one can request the empty feature set, no one can observe what library interface that corresponds to.
+There is no reason to conclude that it *isn't* the empty library.
+
+## RFC [#3146](https://github.com/rust-lang/rfcs/pull/3146)
+
+This more more expensive solution aimed to solve the "running out of good names" problem this has.
+
+Returning to our commutative diagram, instead of
+
+- removing the ⊥ feature downset (this proposal)
+
+- mandating the ⊥ feature downset correspond to ⊥ the empty library interface (first alternative, extra rules on the vertical mappings in diagrams)
+
+We can instead
+
+- allow customizing the mapping from old feature names to new feature names (customizing top horizontal mapping)
+
+This works and allows giving things the "right" names" every time, but is much work work to implement.
+
+I think it might make sense to do this someday, but it is better to start with this.
```

Comment:
Considering the reasons it was rejected, this is unlikely and I would not accept an RFC that implies that this is a future path.

As part of that, please call out the reasons this was rejected

- We need to keep in mind version requirements vs versions. This dealt with version requirement but doesn't work with all version requirements (this can be resolved)
- Whether version requirements or versions, unassisted updates would break builds which is a new semantic I do not think we want (this seems less likely to be resolved)

Reply:
I don't think it was formally rejected --- I withdrew it, but yes your point about manual updates in https://github.com/rust-lang/rfcs/pull/3146#issuecomment-1273473456 was convincing to me and so I would want any new version to reflect that. I will add some text to that effect.

Reply:
Added those caveats in fcf2cc55d1e46443c210cf82b94d0518c407c09f